bcda/main_test.go

```diff
@@ -552,33 +552,15 @@ func (s *MainTestSuite) TestRevokeToken() {
 
 	assert := assert.New(s.T())
 
-	// Create a token
-	userUUID := "82503A18-BF3B-436D-BA7B-BAE09B7FFD2F"
-	tokenString, err := createAccessToken(userUUID, "")
-	assert.Nil(err)
-
 	buf := new(bytes.Buffer)
 	s.testApp.Writer = buf
 
 	// Negative case - attempt to revoke a token passing in a blank token string
 	args := []string{"bcda", "revoke-token", "--access-token", ""}
-	err = s.testApp.Run(args)
+	err := s.testApp.Run(args)
 	assert.Equal("Access token (--access-token) must be provided", err.Error())
```

Comment:
Should this CLI just return an error directly or possibly be removed?  I am not sure why we would run just this test and not just erase the whole CLI.  If we do leave it, I think we should ensure that it returns the expected errors.

Name:     "revoke-token",
			Category: "Authentication tools",
			Usage:    "Revoke an access token",

Reply:
The CLI should continue to function with Okta as the provider, but `main_test.go` isn't the right place to require Okta-specific environmental variables to be set.  Also, later on we should be re-implementing at least this feature for Alpha auth.

Reply:
That makes sense.  But this test sets the provider to alpha, what is the expected behavior if someone runs this CLI with the alpha auth provider?  Can we make sure we get the expected result, even if it is an error?